vevestaX/vevesta.py

```diff
@@ -302,9 +303,49 @@ def __profilingReport(self, fileName):
              }
         ]
         profilingDataframe = pandas.DataFrame(data)
+
+        profileOfVariableDataframe = pandas.DataFrame(
+            {"Field Name": ["Distinct", "Distinct (%)", "Missing", "Missing (%)", "Infinite", "Infinite (%)", "Mean",
+                            "Minimum", "Maximum", "Zeros", "Zeros (%)", "Negative", "Negative (%)",
+                            "Total Memory Size"]})
+
+        for col in self.__data.columns:
+            if self.__data[col].dtype in ["int64", "float64"]:
```

Comment:
use something like this (type(temp[i]) in [int,float] or isinstance(temp[i], (int, float)))}}

Reply:
it can also be int32, double...i dont think what you have written is correct